### PR TITLE
Deploy MM Chart and apply basic tests and teardown

### PR DESCRIPTION
**Pull request by cpanato**
fixes: https://mattermost.atlassian.net/browse/MM-11057

### CODE REVIEW

File: mattermost-helm/values.yaml [hunk 1]
```diff
@@ -171,7 +167,7 @@ minio:
   secretKey: mattermostadmin
   nodeSelector: {}
   persistence:
-    enabled: true
+    enabled: false
```
Comment (crspeller) [inline mattermost-helm/values.yaml]: We should have reasonable defaults in the `values.yaml` not defaults setup for our build server. We should modify this just for the test.

Comment (cpanato) [reply mattermost-helm/values.yaml]: Ok, but in my opinion, all persistence should set to false. 2/5. I will revert that and set in the test for now

**Comment by cpanato** [comment]
@jwilander PTAL

File: mattermost-helm/values.yaml (hunk 1 above)
Comment (crspeller) [reply mattermost-helm/values.yaml]: Ok. But this PR is about setting up the build, not changing defaults, so you can see why I thought it was related.

**Comment by crspeller** [comment]
@cpanato Doesn't seem to be building.

**Comment by cpanato** [comment]
was missing one thing :(. now it is fixed

**Pull request merged**